RadixWallet/Features/HomeFeature/Children/AccountRow/Home+AccountRow+Reducer.swift

```diff
@@ -56,6 +57,10 @@ extension Home {
 				.send(.delegate(.openDetails))
 			case .securityProblemsTapped:
 				.send(.delegate(.openSecurityCenter))
+			case let .accountLockerClaimTapped(details):
+				.run { _ in
+					try await accountLockersClient.claimContent(details)
```

Comment:
we dont care about errors? maybe the client logs errors? otherwise we might wanna log errors here?

Reply:
Adding some logs here just to consider the case the given method throws an error. However, this won't be the case for the edge case discussed last Friday, where user attempts to claim an outdated locker (which no longer has the resources available). In such case, the failure would be at TransactionReview level, which will add logs as the following.

```
2024-09-09T12:31:47+0200 🟥 error com.radixpublishing : Transaction preview failed, error: failedToPrepareTXReview(Radix_Wallet_Dev.TransactionFailure.FailedToPreviewTXReview.failedToRetrieveTXReceipt("ApplicationError(VaultError(ResourceError(InsufficientBalance { requested: 3, actual: 1 })))"))
2024-09-09T12:31:47+0200 🟥 error com.radixpublishing : An error occurred: TransactionReviewFailure(underylying: Radix_Wallet_Dev.TransactionFailure.failedToPrepareTXReview(Radix_Wallet_Dev.TransactionFailure.FailedToPreviewTXReview.failedToRetrieveTXReceipt("ApplicationError(VaultError(ResourceError(InsufficientBalance { requested: 3, actual: 1 })))")))
```